test/test-timer.c

```diff
@@ -161,6 +161,7 @@ TEST_IMPL(timer_init) {
 
   ASSERT(0 == uv_timer_init(uv_default_loop(), &handle));
   ASSERT(0 == uv_timer_get_repeat(&handle));
+  ASSERT(1844674407178900000 < uv_timer_get_due_in(&handle));
```

Comment:
```suggestion
  ASSERT_LE(1844674407178900000ULL, uv_timer_get_due_in(&handle));
```
Although comparing against arbitrary constants is almost certainly wrong because timers don't use wall clock time. The starting point is some arbitrary point in the past - could be last second, could be last century.